cs.c

```diff
@@ -110,7 +110,7 @@ cs_vsnprintf_t cs_vsnprintf = NULL;
 #endif
 
 CAPSTONE_EXPORT
-unsigned int cs_version(int *major, int *minor)
+unsigned int CAPSTONE_API cs_version(int *major, int *minor)
```

Comment:
That would be prettier, but this order is a standard in API definitions on WIndows, for example, https://msdn.microsoft.com/en-us/library/windows/desktop/ms645505(v=vs.85).aspx

Also MSVC requires to put calling convention after a return-type, or else it gives compile error C2059. 

    return-type __stdcall function-name[(argument-list)]

https://msdn.microsoft.com/en-us/library/zxk0tw93.aspx